### DISCUSSION TITLE
[SUGGESTION] Change binary string lists to BOOLEAN type instead.

### DISCUSSION THREAD

**Discussion by HerlanderCarvalho** [Fannovel16/comfyui_controlnet_aux#449] [Ideas]
I've made the code change locally, to fit my needs, but I think it would benefit interoperability with other nodes for everyone, if instead of "enable/disable" on the OpenPose related nodes, you could change it to BOOLEAN instead, so nodes can be changed with general switch nodes. That way I can just use a single switch if I need to.

Thanks!

**Comment by Fannovel16** [comment]
@HerlanderCarvalho Yes a change like that is very simple to made but I don't sure if it'll break backward compatible to older workflows
